Tests/Unit/MiniApp+StringTests.swift

```diff
@@ -8,24 +8,24 @@ class MiniAppStringTests: QuickSpec {
         describe("Mini app + String extension") {
             context("when localized key is passed that is available in Pod bundle") {
                 it("will return valid localized string") {
-                    let okLocalizedTitle = "Ok_title".localizedString()
+                    let okLocalizedTitle = "miniapp.sdk.alert.title.ok".localizedString()
```

Comment:
Why not `MiniAppLocalizable.localize(.ok)` 🤔

Reply:
Yeah, AppCode refactoring is so good that I did not pay too much attention to the unit tests :) I will update this